<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trieicon">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/UP.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="trier">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/blue button.png);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>par defaut</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/statistique.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Statistiques Des affaires juridiques par mois</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajout_icon">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/+.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="profil">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/profile.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/recources/img/affichage.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/recherche.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>430</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/description.png);</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QPushButton" name="description_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/Eye.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="modify">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/gear.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/trash.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/PDF.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="qrcode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/recources/img/qr.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="location">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/geolocalisation.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Menuajout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/menu ajout.png);
background-color: rgb(150, 150, 150);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="leave">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/X.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Formulaire">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 14pt &quot;Arial Rounded MT Bold&quot;;
</string>
     </property>
     <property name="text">
      <string>Formulaire</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="NOM_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="CODE_text">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>CODE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Nom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Code">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="Date">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="cause_text">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Cause</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="date">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cause">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="type_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="classe_text">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Classe</string>
     </property>
    </widget>
    <widget class="QComboBox" name="classe">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="description_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="description">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);
border-image: url(:/recources/img/description.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="done">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>460</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/recources/img/blue button.png);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>terminer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="type">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Violation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Misdemeanors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Felonies</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/blue button.png);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="Qtab">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>721</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Liste_Affaires_Juridique">
     <attribute name="title">
      <string>Liste des affaires juridiques</string>
     </attribute>
     <widget class="QTableView" name="Affichage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Suppression">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Supprimer une affaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="CODE_text_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Code :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Code_supp">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom_supp">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="NOM_text_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ou</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Modification">
     <attribute name="title">
      <string>Modification</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Modifier une affaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="CODE_text_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Code :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Code_mod">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Modifier">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recources/img/blue button.png);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>trieicon</zorder>
   <zorder>trier</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>ajout_icon</zorder>
   <zorder>profil</zorder>
   <zorder>recherche</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>supprimer</zorder>
   <zorder>Qtab</zorder>
   <zorder>Modifier</zorder>
   <zorder>label_15</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
